extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+
+### Statistics
+
+Statistics provide aggregate information about entities in a tileset. Statistics are provided on a per-class basis.
+
+* `count` is the number of entities that conform to the class
+* `properties` contains statistics about property values
+
+Properties have the following built-in statistics:
+
+Name|Description|Type
+--|--|--
+`min`|The minimum property value|Numeric types or fixed-length arrays of numeric types
+`max`|The maximum property value|...
+`mean`|The arithmetic mean of the property values|...
+`median`|The median of the property values|...
+`standardDeviation`|The standard deviation of the property values|...
+`variance`|The variance of the property values|...
+`sum`|The sum of the property values|...
+`occurrences`|Number of enum occurrences|Enums or fixed-length arrays of enums
+
+Tileset authors may define their own additional semantics, like `mode` in the example below.
+
+```jsonc
+{
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "building": {
+            "properties": {
+              "height": {
+                "type": "FLOAT32"
+              },
+              "owners": {
+                "type": "ARRAY",
+                "componentType": "STRING"
+              },
+              "buildingType": {
+                "type": "ENUM",
+                "enumType": "buildingType"
+              }
+            }
+          }
+        },
+        "enums": {
+          "buildingType": {
+            "valueType": "UINT16",
+            "values": [
+              {
+                "name": "Residential",
+                "value": 0
+              },
+              {
+                "name": "Commercial",
+                "value": 1
+              },
+              {
+                "name": "Hospital",
+                "value": 2
+              },
+              {
+                "name": "Other",
+                "value": 3
+              }
+            ]
+          }
+        }
+      },
+      "statistics": {
+        "classes": {
+          "building": {
+            "count": 100000,
+            "properties": {
+              "height": {
+                "min": 3.9,
+                "max": 341.7,
+                "mode": 5.0
+              },
+              "buildingType": {
+                "occurrences": {
+                  "Residential": 50000,
+                  "Commercial": 40950,
+                  "Hospital": 50
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Semantics
+
+By default properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. The full list of built-in semantics can be found in the [Cesium Metadata Semantic Reference](../../../specification/Metadata/Semantics).
```

Comment:
Semantics should have a consistent definition here, #455, and #456 - and we should define more than "interpreted", e.g., is it runtime behavior or just more rich type information?  I think it will vary based on the semantic itself and that specific runtime behavior should be limited.